CONTRIBUTING.md

```diff
@@ -106,6 +106,8 @@ Run the following steps:
 1. The backend uses various configuration settings. You can find them in the `config`
    directory.
    Please copy the file `config_template.yaml` to `config.yaml` and adjust the values.
+   Specify the admin username replacing the value at the key `initial:admin`.
+   Specify the url to the Prometheus server at the key `prometheus:url`.
```

Comment:
These two values are already set in the `config_template.yaml` (https://github.com/DSD-DBS/capella-collab-manager/blob/update-documentation/backend/config/config_template.yaml#L124-L125). Could you please check you local configuration?

Reply:
You can just leave it admin, then you can login with username "admin" :) 
It is somehow documented here: https://github.com/DSD-DBS/capella-collab-manager/blob/staging/doc/getting_started.md